content/blog/gdpr-cheat-sheet.en.md

```diff
@@ -0,0 +1,139 @@
+* Political opinions
+* Religious or philosophical beliefs
+* Trade union membership
+* Genetic data
+* Biometric data for the purpose of uniquely identifying a natural person
+* Data concerning health
+* Data concerning a natural person’s sex life or sexual orientation
+
+### Data subject
+
+The data subject is the person whose [personal data](#personal-data) are [processed](#processing). As a consumer, this usually refers to you.
+
+### Data Minimisation
+
+Data minimisation is an important principle for data protection that the GDPR prescribes. The idea behind it is simple: Only the data necessary for carrying out the stated purpose should be collected and [processed](#processing). If a company wants to send you a newsletter, all it needs is your email address. Following the data minimisation principle, it should not collect any other data.
+
+This concept is important not least because of security reasons. We live in a time where users' private data are regularly leaked to the public because companies have not sufficiently secured them. Data that a company does not have cannot be lost.
+
+### Data Protection Officer
+
+A data protection officer's task is to ensure that a company (or another organisation, such as an authority) complies with data protection law. He is appointed by the company itself and must have a certain degree of independence in order to perform his duties.
+
+If a [controller](#data-controller) has appointed a data protection officer, this person should be your contact person for data protection-related enquiries, for example if you want to exercise your [rights](#rights).
+
+### One Stop Shop Principle
+
+The [One Stop Shop Principle]({{< ref "supervisory-authorities" >}}#one-stop-shop) regulates the responsibility of the [data protection supervisory authorities](#supervisory-authority). For you as a consumer, this means that you may also lodge complaints with the supervisory authority in your own country (in concrete terms, this is the “Member State of [your] habitual residence, place of work or place of the alleged infringement”). This is particularly helpful if the controller is located in another country.
+
+Our [Data Protection Supervisory Authority Finder]({{< ref "supervisory-authorities" >}}#finder) will help you find the supervisory authority responsible for you.
+
+### Personal Data
+
+The concept of personal data is the crux of the GDPR. Almost all of the regulations refer to it. As soon as information can be assigned to a specific person, it is considered personal data. A name is not necessarily required for the assignment. For example, an IP address, a user name, a cookie, or an identification number would suffice.
+
+The information that the IP address 31.41.59.26 called our [generator]({{< ref "generator" >}}) on 25 May 2018 would therefore be considered personal data. On the other hand, the information that on the same day in a bakery a third of the customers paid with cash is not.
```

Comment:
Might just be me but I would associate calling with using a phone. :D How about `visited`?

Reply:
`visited` would definitely work better, I was thinking too technically. :P